src/main/docs/guide/aop/validation.adoc

```diff
@@ -2,12 +2,6 @@ Validation advice is one of the most common advice types you are likely to want
 
 Validation advice is built on https://beanvalidation.org/2.0/spec/[Bean Validation JSR 380], a specification of the Java API for bean validation which ensures that the properties of a bean meet specific criteria, using `javax.validation` annotations such as `@NotNull`, `@Min`, and `@Max`.
 
-Micronaut provides native support for the `javax.validation` annotations with the `micronaut-validation` dependency:
+https://micronaut-projects.github.io/micronaut-validation/snapshot/guide/[Micronaut Validation] provides native support for the `javax.validation` annotations.
 
-dependency::micronaut-validation[]
```

Comment:
why delete this?

Reply:
It is less error-prone to point people to the Micronaut Validation guide, where we explain which dependencies are necessary. I don't want to maintain the same docs in multiple places.